tests/test_printer.py

```diff
@@ -587,7 +587,7 @@ def test_generic_format():
     parser = Parser(ctx, prog)
     module = parser.parse_op()
 
-    assert_print_op(module, expected, None, False)
+    assert_print_op(module, expected, None, print_generic_format=False)
```

Comment:
Could we also change the `diagnostic` argument, either to default `None` or keyword only (or both).